src/main/java/io/github/bobocodebreskul/context/registry/BringContainer.java

```diff
@@ -51,6 +52,8 @@ public static BringContainer run(String... scanPackages) {
 
     definitionRegistry.getBeanDefinitions().forEach(beanDefinition -> container.getBean(beanDefinition.getName()));
 
+    AppConfig.loadProperties();
```

Comment:
It's better to init properties at the start of this method, for cases if we will wont to use them during container initialization.